tools/ros2_dependencies.repos

```diff
@@ -19,3 +19,7 @@ repositories:
     type: git
     url: https://github.com/ros-perception/vision_opencv.git
     version: ros2
+  SteveMacenski/slam_toolbox:
+    type: git
+    url: https://github.com/SteveMacenski/slam_toolbox.git
+    version: ros2
```

Comment:
I'm pretty sure that's not a real branch - try `foxy-devel`

Reply:
Got it, thanks